src/utils/ObjectUtils.js

```diff
@@ -1,4 +1,5 @@
 import _ from 'underscore';
+import isPlainObject from './object/isPlainObject';
```

Comment:
use https://www.npmjs.com/package/lodash.isplainobject

Reply:
Saw my comment in the description? https://github.com/mixmaxhq/backbone-publication/pull/16#issue-254729419

Reply:
This actually required to add commonjs and nodeResolve plugins of rollup to get it working properly. thanks @wearhere for the instructions on slack!